ccm_web/server/src/canvas/canvas.controller.ts

```diff
@@ -32,7 +32,7 @@ export class CanvasController {
     }
 
     if (user.canvasToken !== null) {
-      logger.debug(`User ${user.loginId} already has a CanvasToken record; redirecting to home page...`)
+      logger.warn(`User ${user.loginId} already has a CanvasToken record; redirecting to home page...`)
       return res.redirect('/')
     }
```

Comment:
Deciding to leave this for now as it's possible the user could navigate directly to `canvas/redirectOAuth` and we'd want to bail if they have a token in that case.